packages/data-provider/src/services/categoryService.ts

```diff
@@ -18,15 +18,17 @@ const categoryService = (dependencies: BaseDependencies) => {
 
 				return {
 					id: category.id,
-					dictionary: {
-						name: category.activeDictionary?.name || '',
-						version: category.activeDictionary?.version || '',
-					},
+					...(category.activeDictionary && {
+						dictionary: {
+							name: category.activeDictionary.name,
+							version: category.activeDictionary.version,
+						},
+					}),
```

Comment:
This is correct but was hard for me to read, would you consider instead:
```suggestion
					dictionary: category.activeDictionary
						? {
								name: category.activeDictionary.name,
								version: category.activeDictionary.version,
							}
						: undefined,
```

Reply:
formatting code as suggested.